lib/grape/api/instance.rb

```diff
@@ -191,6 +191,9 @@ def cascade?
       # with a list of HTTP methods that can be called. Also add a route that
       # will return an HTTP 405 response for any HTTP method that the resource
       # cannot handle.
+      #
+      # rubocop:disable Metrics/AbcSize because of the complex workflow
```

Comment:
Disable these in .rubocop.yml or just run `rubocop -a ; rubocop --auto-gen-config`.